features/overlijden-indicatie-overleden.feature

```diff
@@ -142,4 +142,30 @@ Functionaliteit: overlijden: indicatie overleden
       | groep overlijden     | 060800                  |
       | datum overlijden     | 060810                  |
       | plaats overlijden    | 060820                  |
-      | land overlijden      | 060830                  |
\ No newline at end of file
+      | land overlijden      | 060830                  |
+
+  Rule: Onjuiste gegevens worden niet geleverd
```

Comment:
Ik denk niet dat dit de goede feature is om deze scenario toe te voegen. In dit feature file wordt de indicatie overleden functionaliteit beschreven en daar heeft onjuiste gegevens in overlijden niets mee te maken (denk ik)

Reply:
Scenario verwijderd